src/org/camunda/latera/bss/testting/TestExecution.groovy

```diff
@@ -0,0 +1,30 @@
+package org.camunda.latera.bss.testing;
+
+import org.camunda.bpm.engine.impl.pvm.runtime.ExecutionImpl
+import org.camunda.bpm.engine.runtime.Incident
+import org.yaml.snakeyaml.Yaml
+
+class TestExecution extends ExecutionImpl {
+  public LinkedHashMap config
+  public String testName
+
+  TestExecution(String testName) {
+    Yaml parser = new Yaml()
+    this.config = parser.load(("/camunda/lib/execution.yml" as File).text)
+    this.testName = testName
+  }
+
+  public Object getVariable(String variableName) {
```

Comment:
Don't see setVariable(CharSequence,def) method implementation, but there are some classes which use it (Order, HOMS).